src/main/java/raisetech/studentManagement/service/StudentService.java

```diff
@@ -25,4 +28,23 @@ public List<Student> searchStudentList() {
   public List<StudentsCourses> searchStudentCourseList() {
     return repository.searchStudentCourse();
   }
+
+  @Transactional
+  public void registerStudent(StudentDetail studentDetail){
+    Random random = new Random();
+
+    //ランダムなstudentのid
+    String randomStudentId = String.valueOf(random.nextInt(100000000));
+    // 生徒情報
+    studentDetail.getStudent().setId(randomStudentId);
+    studentDetail.getStudent().setDeleted(false);
+    repository.insertStudent(studentDetail.getStudent());
+    //コース情報
+    for(StudentsCourses studentsCourses : studentDetail.getStudentsCourses()){
+    studentsCourses.setStudentId(randomStudentId);
+    studentsCourses.setId(String.valueOf(random.nextInt(100000000)));
+    repository.insertStudentCourse(studentsCourses);
+    }
```

Comment:
フォーマットしてれば以下のようになるはずです。
コードフォーマットを実施してみましょう 👍 
```suggestion
    for(StudentsCourses studentsCourses : studentDetail.getStudentsCourses()){
      studentsCourses.setStudentId(randomStudentId);
      studentsCourses.setId(String.valueOf(random.nextInt(100000000)));
      repository.insertStudentCourse(studentsCourses);
    }
```